commitizen/defaults.py

```diff
@@ -153,3 +154,29 @@ def get_tag_regexes(
         **{f"${k}": v for k, v in regexs.items()},
         **{f"${{{k}}}": v for k, v in regexs.items()},
     }
+
+
+def __getattr__(name: str) -> Any:
+    """Deprecated variable access"""
+    deprecated_vars = {
```

Comment:
```suggestion
    # {"deprecated key": (value, "new key")}
    deprecated_vars = {
```

I don't really have a better name for it. so let's add a comment here